src/app/foodLogController.java

```diff
@@ -0,0 +1,41 @@
+package app;
+
+import javafx.scene.control.Button;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Label;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+
+/**
+ *
+ * @author aswecker
+ */
+public class foodLogController {
+    /**
+     * The date picker for the date a user is logging food
+     */
+    private DatePicker startDatePicker;
```

Comment:
Please add the @FXML annotation right above the method name. I can explain that later.